src/stylesheets/components/_quit-survey-modal.scss

```diff
@@ -0,0 +1,78 @@
+.quit-survey-modal {
+  &__container {
+    position: fixed;
+    top: 0;
+
+    width: 100%;
+    height: 100%;
+
+    background-color: rgba(0, 0, 0, 0.4);
+  }
+
+  &__content {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+
+    display: flex;
+    flex-flow: column;
+
+    width: rem(467px);
+    height: rem(178px);
+    padding: rem(24px);
+
+    background: map_get($brand-black, 'primary');
+    border-radius: rem(14px);
+    transform: translate(-50%, -50%);
+  }
+
+
+  &__header {
+    display: flex;
+    margin-bottom: rem(8px);
+
+    color: map_get($brand-white, 'primary');
+    font-weight: map_get($font-weight, 'bolder');
+    font-size: map_get($font-sizes, 'xl');
+    line-height: rem(28px);
+  }
+
+  &__body {
+    display: flex;
+
+    color: map_get($brand-white, 'primary');
+    font-size: map_get($font-sizes, 'md');
+    line-height: rem(22px);
+
+    opacity: 0.7;
+  }
+
+  &__footer {
+    display: flex;
+    flex: 1 1 auto;
+    align-items: flex-end;
+
+    width: rem(188px);
+    margin-left: auto;
+  }
+
+  &__yes-button {
+    @include warning-actions();
```

Comment:
@rafayet-monon Instead of adding mixin for both yeas and cancel buttons how about adding a `modal` class and set the share style on the button within the class

```scss
.modal {
  .button {
    width: rem(90px);
    height: rem(40px);

    font-weight: map_get($font-weight, 'bolder');
  }
}
```

then on HTML side just add the `modal` class to the modal